pdart/astroquery/Astroquery.py

```diff
@@ -17,7 +17,7 @@
 
 _YMD = Tuple[int, int, int]
 
-_ACCEPTED_INSTRUMENTS: str = "IJNU"
+_ACCEPTED_INSTRUMENTS: str = "IJLNOUWXYZ"
```

Comment:
I was sloppy before and wrote "IJU" in other places in the code.  Please refactor to fix my sloppiness so we only have one source of truth.  You'll want to rename `_ACCEPTED_INSTRUMENTS` to `ACCEPTED_INSTRUMENTS` so it's no longer private and other modules can import it.  See other comments where it should be used.